src/content/guides/production.md

```diff
@@ -18,17 +18,18 @@ contributors:
   - mehrdaad
   - SevenOutman
   - AnayaDesign
+  - wizardofhogwarts
   - aholzner
 ---
 
-In this guide we'll dive into some of the best practices and utilities for building a production site or application.
+In this guide, we'll dive into some of the best practices and utilities for building a production site or application.
 
 T> This walkthrough stems from [Tree Shaking](/guides/tree-shaking) and [Development](/guides/development). Please ensure you are familiar with the concepts/setup introduced in those guides before continuing on.
 
 
 ## Setup
 
-The goals of _development_ and _production_ builds differ greatly. In _development_, we want strong source mapping and a localhost server with live reloading or hot module replacement. In _production_, our goals shift to a focus on minified bundles, lighter weight source maps, and optimized assets to improve load time. With this logical separation at hand, we typically recommend writing __separate webpack configurations__ for each environment.
+The goals of _development_ and _production_ build differ greatly. In _development_, we want strong source mapping and a localhost server with live reloading or hot module replacement. In _production_, our goals shift to a focus on minified bundles, lighter weight source maps, and optimized assets to improve load time. With this logical separation at hand, we typically recommend writing __separate webpack configurations__ for each environment.
```

Comment:
I still think that `development build and production build` becomes `builds`.
If it was saying: `the difference between those builds` you wouldn't change it to `build` right?

I did take a look at this: https://www.grammarbook.com/grammar/subjectVerbAgree.asp
But still cannot see why you want to change that 😅

Reply:
I was talking in context of goals. But no worries if you cross checked it. I may revert it if u say.

Reply:
or easier apply suggest 
```suggestion
The goals of _development_ and _production_ builds differ greatly. In _development_, we want strong source mapping and a localhost server with live reloading or hot module replacement. In _production_, our goals shift to a focus on minified bundles, lighter weight source maps, and optimized assets to improve load time. With this logical separation at hand, we typically recommend writing __separate webpack configurations__ for each environment.
```